lib/ansible/modules/network/cloudengine/ce_static_route.py

```diff
@@ -0,0 +1,845 @@
+                if static_route["nexthop"] and self.next_hop:
+                    if static_route["prefix"].lower() == self.prefix.lower() \
+                            and static_route["maskLength"] == self.mask \
+                            and static_route["nexthop"].lower() == self.next_hop.lower() \
+                            and static_route["afType"] == version:
+                        change = True
+                        return change
+                if static_route["ifName"] and self.nhp_interface:
+                    if static_route["prefix"].lower() == self.prefix.lower() \
+                            and static_route["maskLength"] == self.mask \
+                            and static_route["ifName"].lower() == self.nhp_interface.lower() \
+                            and static_route["afType"] == version:
+                        change = True
+                        return change
+                else:
+                    continue
+            change = False
+        return change
+
+    def get_proposed(self):
+        """get proposed information"""
+
+        self.proposed['prefix'] = self.prefix
+        self.proposed['mask'] = self.mask
+        self.proposed['afType'] = self.aftype
+        self.proposed['next_hop'] = self.next_hop
+        self.proposed['ifName'] = self.nhp_interface
+        self.proposed['vrfName'] = self.vrf
+        self.proposed['destVrfName'] = self.destvrf
+        if self.tag:
+            self.proposed['tag'] = self.tag
+        if self.description:
+            self.proposed['description'] = self.description
+        if self.pref is None:
+            self.proposed['preference'] = 60
+        else:
+            self.proposed['preference'] = self.pref
+        self.proposed['state'] = self.state
+
+    def get_existing(self):
+        """get existing information"""
+
+        change = self.get_ip_static_route()
+        self.existing['sroute'] = self.static_routes_info["sroute"]
+        self.changed = bool(change)
+
+    def get_end_state(self):
+        """get end state information"""
+
+        self.get_static_route(self.state)
+        self.end_state['sroute'] = self.static_routes_info["sroute"]
+
+    def work(self):
+        """worker"""
+
+        self.check_params()
+        self.get_existing()
+        self.get_proposed()
+        self.set_ip_static_route()
+        self.set_update_cmd()
+        self.get_end_state()
+        self.results['changed'] = self.changed
+        self.results['proposed'] = self.proposed
+        self.results['existing'] = self.existing
+        self.results['end_state'] = self.end_state
+        if self.changed:
+            self.results['updates'] = self.updates_cmd
+        else:
+            self.results['updates'] = list()
+
+        self.module.exit_json(**self.results)
+
+
+def main():
+    """main"""
+
+    argument_spec = dict(
+        prefix=dict(required=True, type='str'),
+        mask=dict(required=True, type='str'),
```

Comment:
For integers please:
use `type='int`
remove the `isdigit` check from `check_params`